bynder_sdk/util.py

```diff
@@ -11,6 +21,11 @@ def parse_json_for_response(response):
 
 class SessionMixin:
```

Comment:
Since this mixin is used on top of [request's Session](https://requests.readthedocs.io/en/master/user/advanced/#session-objects) object I believe you should be able to achieve the same result by updating the session's headers like this:

```suggestion
class SessionMixin:
    def __init__(self):
        super().__init__()
        self.headers.update(UA_HEADER)
```

The PermanentToken implementation does the same:
https://github.com/Bynder/bynder-python-sdk/blob/06c000f98a9cb13ff709c0b5308f20024b5f069c/bynder_sdk/permanent_token.py#L6-L13